structs.go

```diff
@@ -20,6 +20,9 @@ type Connection struct {
 	// Client is the http client used to make requests, allowing settings things
 	// such as timeouts etc
 	Client *http.Client
+
+	// Specify to use numbers when unmarshal
+	UseNumber bool
```

Comment:
Since there is a method to enable this, it should probably be non-exported, otherwise it might as well not be a method and just specify that users should set this attribute directly.